bundles/org.openhab.binding.irobot/src/main/java/org/openhab/binding/irobot/internal/handler/RoombaHandler.java

```diff
@@ -189,9 +189,38 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                         String[] params = cmds[1].split(";");
 
                         String mapId = params[0];
-                        String[] regionIds = params[1].split(",");
-
-                        MQTTProtocol.Request request = new MQTTProtocol.CleanRoomsRequest("start", mapId, regionIds);
+                        String user_mapv_id;
+                        if (params.length >= 3) {
+                            user_mapv_id = params[2];
+                        } else {
+                            user_mapv_id = null;
+                        }
+
+                        String[] ArrayRegions = params[1].split(",");
```

Comment:
Naming convention: `arrayRegions` instead of `ArrayRegions`.
`regions` would even be better.